readthedocs_build/config/parser.py

```diff
@@ -0,0 +1,27 @@
+import yaml
+
+
+__all__ = ('parse', 'ParseError')
+
+
+class ParseError(Exception):
+    pass
+
+
+def parse(stream):
+    """
+    Take file-like object and return a list of project configurations.
+
+    The files need be valid YAML and only contain mappings as documents.
+    Everything else raises a ``ParseError``.
+    """
+    try:
+        configs = list(yaml.load_all(stream))
```

Comment:
`load_all` can run code from the YAML file ([docs](http://pyyaml.org/wiki/PyYAMLDocumentation#LoadingYAML)) - you probably want `safe_load_all`.

If this only runs inside the build container, it doesn't matter so much, but unless there's a reason to allow the unsafe parts, it's probably worth using the safe variant anyway.